site/src/util/mdx-utils.tsx

```diff
@@ -3,11 +3,10 @@ import matter from "gray-matter";
 import { MDXRemoteSerializeResult } from "next-mdx-remote";
 import { serialize } from "next-mdx-remote/serialize";
```

Comment:
This was impossible without #322 because `next-mdx-remote/serialize` v2 is ESM only. The build script was  crashing.